test/EasyRdf/ContainerTest.php

```diff
@@ -40,13 +40,13 @@
 
 class ContainerTest extends TestCase
 {
-    public function setUp()
+    public function setUp(): void
```

Comment:
Is adding the void return type a PHP style thing? Has PHP 8 started emitting warnings if it is absent?

Reply:
Its because of PHPUnit. If I remember correctly, PHPUnit changed `setUp` to return `void`. If our `setUp` doesn't, we will get a declaration error.